native/src/main/java/io/ballerina/stdlib/java.jms/JmsConnection.java

```diff
@@ -234,18 +232,16 @@ public static Object stop(BObject connection) {
      *                  to close a socket connection can cause this exception to be thrown.
      */
     public static Object close(BObject connection) {
-        Object nativeConnection = connection.getNativeData(NATIVE_CONNECTION);
-        if (Objects.nonNull(nativeConnection)) {
-            try {
-                ((Connection) nativeConnection).close();
-                return null;
-            } catch (JMSException exception) {
-                BError cause = ErrorCreator.createError(exception);
-                return ErrorCreator.createError(ModuleUtils.getModule(), JMS_ERROR,
-                        StringUtils.fromString("Error occurred while closing the connection"), cause, null);
-            }
+        Connection nativeConnection = (Connection) connection.getNativeData(NATIVE_CONNECTION);
+        try {
+            nativeConnection.close();
+        } catch (JMSException exception) {
+            BError cause = ErrorCreator.createError(exception);
+            return ErrorCreator.createError(ModuleUtils.getModule(), JMS_ERROR,
```

Comment:
shall we add a utility function to create jms errors

Reply:
Fixed with:
- [f94f579561ba6f6f299225a9b238dc3ae7fb95b2](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/95/commits/f94f579561ba6f6f299225a9b238dc3ae7fb95b2)
- [8a63496983d3185b57d140a2c4025d63c715f499](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/95/commits/8a63496983d3185b57d140a2c4025d63c715f499)